### PR TITLE
Ar 214

### PR DESCRIPTION
**Pull request by davidedmonds**

### CODE REVIEW

File: src/main/java/dynamok/sink/DynamoDbClient.java [hunk 1]
```diff
@@ -61,13 +67,41 @@ void writeBatch(Collection<SinkRecord> records) throws UnprocessedItemsException
...
+            return config.keyAttributeName;
+        } else {
+            Map<String, String> keyTypeMap = dynamoDB.describeTable(table)
+                    .getTable().getKeySchema()
+                    .stream()
+                    .collect(Collectors.toMap(KeySchemaElement::getKeyType, KeySchemaElement::getAttributeName));
+            if (keyTypeMap.containsKey(KeyType.RANGE.name())) {
+                throw new InvalidTableException(table, keyTypeMap.get(KeyType.RANGE.name()));
+            } else {
+                return keyTypeMap.get(KeyType.HASH.name());
```
Comment (javierholguera) [inline src/main/java/dynamok/sink/DynamoDbClient.java]: Is it worth documenting in the description of `key.attribute.name` that we can infer it automatically for tables that only use partition keys and don't have sort keys?

Comment (davidedmonds) [reply src/main/java/dynamok/sink/DynamoDbClient.java]: We could - if the expectation was that people would generally omit `key.attribute.name` I'd be tempted to remove the config and only use inference though.

Comment (davidedmonds) [reply src/main/java/dynamok/sink/DynamoDbClient.java]: Discussed and agreed we'll remove the config property as the metadata lookup doesn't seem to be too onerous.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+1.1.0
+=====
+- Add support for `String` type keys, where the DynamoDB table only has a `HASH` key. The name of
+  the `HASH` key's attribute may be specified using the `key.attribute.name` property. It may also
```
Comment (javierholguera) [inline CHANGELOG.md]: This need to be updated.

Comment (davidedmonds) [reply CHANGELOG.md]: Done.

**Pull request merged**